doc/api/errors.md

```diff
@@ -1391,13 +1391,13 @@ An invalid or unknown file encoding was passed.
 <a id="ERR_INVALID_PACKAGE_CONFIG"></a>
 ### `ERR_INVALID_PACKAGE_CONFIG`
 
-An invalid `package.json` file was found which failed parsing.
+An invalid [`package.json`][] file was found which failed parsing.
 
 <a id="ERR_INVALID_PACKAGE_TARGET"></a>
 ### `ERR_INVALID_PACKAGE_TARGET`
 
-The `package.json` [exports][] field contains an invalid target mapping value
-for the attempted module resolution.
+The [`package.json`][] [`"exports"`][] field contains an invalid target mapping
```

Comment:
Since the 2nd link already points to the more specific section on the same page, would it make sense to drop the first link to the overall page? Two links right next to each other may look like one link which may be confusing here.